hivemind/utils/tensor_descr.py

```diff
@@ -70,13 +83,48 @@ def from_tensor(cls, tensor: torch.Tensor, compression=CompressionType.NONE) ->
             device=tensor.device,
             requires_grad=tensor.requires_grad,
             pin_memory=_safe_check_pinned(tensor),
-            compression=compression if tensor.is_floating_point() else CompressionType.NONE
+            compression=compression if tensor.is_floating_point() else CompressionType.NONE,
         )
 
     def make_empty(self, *batch_size: int, **kwargs) -> torch.Tensor:
         assert self.shape[0] is None, "Make sure 0-th dimension is not specified (set to None)"
         return super().make_empty(size=(*batch_size, *self.shape[1:]), **kwargs)
 
+    def packb(self) -> bytes:
+        obj_dict = asdict(self)
+        obj_dict.pop("device")
```

Comment:
Let's move this line right below `if self.device is not None:` (currently, reading the code surprised me with removing the device, and I got the reasoning only when I finished reading this function).